requirements.txt

```diff
@@ -28,7 +28,7 @@ python-neutronclient
 python-novaclient
 python-octaviaclient
 python-swiftclient
-tenacity
+tenacity>8.2.0
```

Comment:
This pin seems a bit strange; is it to error out against something else pinning a lower version?

Reply:
I agree it's strange, something is inducing the resolver to pick tenacity-5.5, I don't know why, although this doesn't lead into any errors.

Reply:
> > Is pytest now a requirement for py3.11?
> 
> not a 'requirement', although nose is no longer maintained, see [nose-devs/nose#1118 (comment)](https://github.com/nose-devs/nose/issues/1118#issuecomment-1061887733)

pytest it is then!